src/playwright/transformPlaywrightJson.ts

```diff
@@ -43,7 +43,7 @@ export const transformPlaywrightJson = (src: string) => {
   }
   const stories = Object.values(json.stories) as Story[];
   const titleIdToStories = stories.reduce((acc, story) => {
-    const titleId = toId(story.title);
+    const titleId = toId(story.kind, story.story);
```

Comment:
why is this change needed? also `kind/story` are deprecated and `title/name` should be used instead. but i don't know why we need this change to begin with.

Reply:
This seems to be necessary. Another working version of the code is `toId(story.title, story.name)`. 

```ts
// With SB 6.5 and this code: toId(story.title) 
{ titleId: 'atoms-button' } // Wrong!

// With SB 6.5 and toId(story.kind, story.story) || toId(story.title, story.name)
{ titleId: 'atoms-button--wait-for' } // Correct!
```

Reply:
Done. Replaced by `story.title` and `story.name`